MessengerOnFileForLinux/scripts/check_clang-tidy_job_status.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+CLANG_TIDY_OUTPUT_FILE=$1
+COMMAND=`grep warning $CLANG_TIDY_OUTPUT_FILE | wc -l`
+if [ $COMMAND -eq 0 ] ; then
+    mkdir no_clang-tidy_warnigns
```

Comment:
po co tworzymy ten folder?

Reply:
cd do folderu który nie istnieje wywala joba. Lepszego pomysłu na to nie mam

Reply:
trzeba to inaczej opisac, bo tak jest nieczytelne

Reply:
co proponujesz?

Reply:
nwm, trzeba sie dowiedzieć jak jeszcze wywalic joba w legitny sposob.

Reply:
Dobra, joba sfailuje wszystko co kończy się w skrypcie innym statusem niż 'exit 0'. Proponuje takie rozwiazanie:
if [ $COMMAND -ne 0 ] ; then
    exit 1

Lepszego nic nie wymyslimy, bo w samym skrypcie CI nie ma możliwości robienia warunków i trzeba joba zabić jakoś poprzez skrypt bashowy

Reply:
a może zrób tak:

if ... then
JOB_FAIL := 1;
exit JOB_FAIL